testing/oidc/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:16.2.0-slim
+
+WORKDIR /app
+
+RUN apt update && apt upgrade -qy && apt install -qy curl
+
+COPY package.json ./
+
+RUN  rm -rf node_modules
+
+RUN npm install -g npm@latest && \
+    npm i camelcase oidc-provider
+
+COPY . .
+
+CMD [ "node", "server.js" ]
```

Comment:
Why not use the `ls-aai-mock` from GDI?

Reply:
Good point! The GDI repo does not provide an image for ls-aai-mock, so I thought the current solution is better than pulling that repo and building images as it can be used for integration tests in the future. But, I can see if I can add some README instructions on how to plug in the ls-aai-mock as well for further testing.

Reply:
The only thing you need is in the ls-aai-mock docker compose file.